addons/website_mail_channel/static/src/snippets/s_channel/000.js

```diff
@@ -1,11 +1,19 @@
-odoo.define('website_mail_channel.snippet', function (require) {
```

Comment:
@rdeodoo Note: for files you split, there is nothing you can do since it requires a merge I may or not do by myself during one night. But for files you move then slightly modify, you should make the move in a separate commit. Here GitHub is intelligent enough to know it is a move but our forward-port bot won't be ;) I'll do it by myself here since already cherry-picking ;)